maquetado/scss/styles.scss

```diff
@@ -9,6 +9,45 @@
   width: 1000px;
 }
 
+.big-container {
+  padding: 10px;
+}
+
+.books-list-container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
```

Comment:
Same, use `column` class for this. This way you don't need to define `display: flex` again and again. Remember the alphabetical order